docs/ImagingApi.md

```diff
@@ -872,6 +872,36 @@ Name | Type | Description  | Notes
 
 [[Back to top]](#) [[Back to API list]](API_README.md#documentation-for-api-endpoints) [[Back to Model list]](API_README.md#documentation-for-models) [[Back to API_README]](API_README.md)
 
+<a name="createwebsiteimagefeatures"></a>
+## **CreateWebSiteImageFeatures**
+> void CreateWebSiteImageFeatures(CreateWebSiteImageFeaturesRequest request)
+
+Extract images features from web page and add them to search context
+
+### **CreateWebSiteImageFeaturesRequest** Parameters
+```csharp
+CreateWebSiteImageFeaturesRequest(
+    string searchContextId, 
+    string imagesSource, 
+    System.IO.Stream imageData = null, 
```

Comment:
We don`t use parameter System.IO.Stream imageData in the CreateWebSiteImageFeatures method.